cloud/main.js

```diff
@@ -237,7 +237,11 @@ Parse.Cloud.define("joinQueue", function(request, response) {
                   }
                 }
               ).then( function(){
-                response.success(player);
+                sendDataChangedNotification().then(function() {
+                  response.success(player);
+                }).fail(function(err) {
+                  response.error(err);
+                });
```

Comment:
@giladgo I'm not sure about this. Why would you not respond as "success" if the notification wasn't sent for some reason?